src/Microsoft.VisualStudio.SDK.Analyzers/Microsoft.VisualStudio.SDK.Analyzers.csproj

```diff
@@ -10,10 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.3.1" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="16.9.31004.209" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="16.9.31004.209" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.10.56" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.2.0" PrivateAssets="all" />
```

Comment:
Upgrading the roslyn package versions cut off support for users of these analyzers that run on older VS versions or .NET SDKs. We generally *want* to reference an older than latest version to work for more customers. I think we should back this out.
But most of your new nullable ref annotations that I imagine you added as a result of this upgrade can and probably should stick around, so hopefully you have no need to revert them.

Reply:
Reverted